desktop/cmp/rest/RestGridModel.js

```diff
@@ -174,60 +192,71 @@ export class RestGridModel {
         this.formModel.openClone(clone);
     }
 
-    @action
-    deleteSelection() {
-        const record = this.selectedRecord;
-        if (record) this.deleteRecord(record);
-    }
-
-    @action
     deleteRecord(record) {
         throwIf(this.readonly, 'Record not deleted: this grid is read-only');
         this.store.deleteRecordAsync(record)
             .then(() => this.formModel.close())
             .catchDefault();
     }
 
-    @action
-    editSelection() {
-        const record = this.selectedRecord;
-        if (record) this.editRecord(record);
+    async bulkDeleteRecordsAsync(records) {
+        throwIf(this.readonly, 'Records not deleted: this grid is read-only');
+        const resp = await this.store.bulkDeleteRecordsAsync(records).catchDefault(),
+            intent = resp.fail > 0 ? 'warning' : 'success',
+            message = `Deleted ${resp.success} ${pluralize(this.unit)} with ${resp.fail} failures`;
+
+        XH.toast({intent, message});
     }
 
-    @action
     editRecord(record) {
         this.formModel.openEdit(record);
     }
 
-    @action
```

Comment:
None of these "actions" appear to be modifying any observables... maybe left over decorators from a time when they did? Please correct if wrong